frontend/src/products.tsx

```diff
@@ -337,6 +337,11 @@ export const getTreeItemsNew = (): FileSystemImport[] => [
     { path: `Early access feature`, type: 'early_access_feature', href: urls.earlyAccessFeature('new') },
     { path: `Experiment`, type: 'experiment', href: urls.experiment('new') },
     { path: `Feature flag`, type: 'feature_flag', href: urls.featureFlag('new') },
+    {
+        path: `Insight/Calendar Heatmap`,
+        type: 'insight',
+        href: urls.insightNew({ type: InsightType.CALENDAR_HEATMAP }),
+    },
```

Comment:
FYI: This is actually a generated file. It is a recent change and I am not sure if it was just commited or you had to update it manually, so commenting only for the sake of sharing this.

You've already added it to the `manifest.tsx` so I guess all is good :D https://github.com/PostHog/posthog/blob/fe5e6012b6fb48e5163215e0752c433dc63f217e/products/product_analytics/manifest.tsx#L109-L113